File: HW/HW10/HW10Test.java
import org.junit.Test;

import java.util.ArrayList;
import java.util.List;

public class HW10Test {
    @Test
    public void test() {
        TreeNode t = new TreeNode(4);
        t.left = new TreeNode(3);
        t.left.left = new TreeNode(1);
        t.right = new TreeNode(8);
        t.right.left = new TreeNode(5);
        t.right.right = new TreeNode(9);
        List<List<Integer>> test = new ArrayList<>();
        test = TreeNode.levelOrder(t);
        System.out.println(test);

        TreeNode t1 = null;
        test = TreeNode.levelOrder(t1);
        System.out.println(test);
    }
}

File: HW/HW10/HW10Test.py
import unittest
from TreeNode import *

class MyTestCase(unittest.TestCase):
    def test_HW10(self):
        t = TreeNode(4)
        t.left = TreeNode(3)
        t.left.left = TreeNode(1)
        t.right = TreeNode(8)
        t.right.left = TreeNode(5)
        t.right.right = TreeNode(9)

        test = TreeNode.levelOrder(self, t)
        print(test)

        t1 = None
        test = TreeNode.levelOrder(self, t1)
        print(test)


if __name__ == '__main__':
    unittest.main()
